Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -19,32 +20,41 @@ int main()
     int *p = &x;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: " /* << This will print the address of p/ << endl;
```

Comment:
Two things about this:
1) No need to keep any of your predictions as comments, just print them to console
2) This line will cause your program not to compile, as your comment doesn't end (`*/` vs `/`)